Minor changes to automation script

ned1313 opened the pull request:
Changed cloudadmin to AzureStackAdmin and the Privileged Endpoint to use AzS-ERCS01 instead of 10.10.10.10

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

ned1313 committed:
Minor changes to automation script

Changed cloudadmin to AzureStackAdmin and the Privileged Endpoint to use AzS-ERCS01 instead of 10.10.10.10
